### PR TITLE
feat: add visual status indicators to the UIToggle component

### PR DESCRIPTION
**Pull request by Vladislavs-Silins-SAP**
The following changes were implemented:

1. Added SVG icons `SwitchOff` and `SwitchOn` 
2. Added visual indicators to show switch status in the UIToggle component

While we are at it, we should make toggle switches a bit bigger to better comply with "Target hit area" accessibility topic. They are too small now.

image

### CODE REVIEW

File: packages/ui-components/src/components/UIToggle/UIToggle.tsx [hunk 1]
```diff
@@ -92,13 +166,60 @@ const COLORS = {
...
+     */
+    replaceThumbWithIcon(checked?: boolean) {
+        const isSwitchOn = checked ?? this.props.defaultChecked ?? false;
+        if (this.toggleRootRef.current) {
+            const thumbElement = (this.toggleRootRef.current as HTMLElement)?.querySelector('.ms-Toggle-thumb');
+
+            if (thumbElement) {
+                const style = ICON_STYLE.get(getIconStyleKey(this.props.size ?? UIToggleSize.Standard, isSwitchOn));
+                ReactDOM.render(
+                    <UIIcon iconName={isSwitchOn ? 'SwitchOn' : 'SwitchOff'} style={style} />,
```
Comment (815are) [inline packages/ui-components/src/components/UIToggle/UIToggle.tsx:218]: as there is enum, maybe `'SwitchOn'` -> `UiIcons.SwitchOn`?

**Review by 815are — APPROVED** [review]
Checked:
1. code - easy to understand. As there no any custom renderer available from `flunet-ui` currently do not see better way how to inject icons(alternative maybe font icons or mask-images);
2. test is maintainer;
3. changeset exists


small comment about string value where enum entry can be used

File: packages/ui-components/src/components/UIToggle/UIToggle.tsx (hunk 1 above)
Comment (Vladislavs-Silins-SAP) [reply packages/ui-components/src/components/UIToggle/UIToggle.tsx]: Done.

**Review by 815are — DISMISSED** [review]

**Review by 815are — APPROVED** [review]

**Review by AlinaGovoruhina — APPROVED** [review]
Code looks good, well commented, unit test coverage sufficient, changeset present.

**Pull request merged**